CHANGELOG.md

```diff
@@ -20,7 +20,10 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 * controller, worker: Fix listening on IPv6 addresses
   ([Issue](https://github.com/hashicorp/boundary/issues/701))
   ([PR](https://github.com/hashicorp/boundary/pull/703))
+* dev mode worker: Fix setting controller address for worker in dev mode
```

Comment:
```suggestion
* worker: Fix setting controller address for worker in dev mode
```

Mostly this is to try to keep the number of categories down since we already have `worker`. I'm not sure what level of specificity we want here. Thoughts?